WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -532,6 +538,22 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
                     AnalyticsTracker.track(AnalyticsTracker.Stat.NOTIFICATION_PENDING_DRAFTS_SETTINGS_DISABLED);
                 }
             }
+        } else if (key.equals(getString(R.string.wp_pref_custom_notification_sound))) {
+            final String defaultPath =
+                    getString(R.string.notification_settings_item_sights_and_sounds_choose_sound_default);
+            final String value = sharedPreferences.getString(key, defaultPath);
+
+            if (value.trim().startsWith("file://")) {
```

Comment:
shouldn't we also make a case-insensitive comparison here ?

Reply:
Hmm, I'm under the impression it cannot happen (it's not a direct user-input) but since I can't be sure let's make it certain. Done with 00363c0.